document/fr.opensagres.xdocreport.document.odt/src/test/java/fr/opensagres/xdocreport/document/odt/styling/TestODTStyling.java

```diff
@@ -40,8 +45,7 @@
 import org.xml.sax.XMLReader;
 import org.xml.sax.helpers.XMLReaderFactory;
 
-import com.sun.org.apache.xml.internal.serialize.OutputFormat;
-import com.sun.org.apache.xml.internal.serialize.XMLSerializer;
+import com.sun.org.apache.xml.internal.serializer.OutputPropertiesFactory;
```

Comment:
I would like to avoid having dependencies to internal classes.

Reply:
This commit is one of the two commits I consider a hack. The other is where I touch this class again.

Though I have improved the situation, from 2 imports of internal class to 1 import of internal classes.

And also this is a test class.

Reply:
You are right, thanks for the clarification. Could you squask your commits in one commit please.

Reply:
To be clear; squash all the commits in the PR into one commit.

Reply:
yes please.